specs/phase0/p2p-interface.md

```diff
@@ -355,17 +355,20 @@ to subscribing nodes (typically validators) to be included in future blocks.
 
 The following validations MUST pass before forwarding the `signed_aggregate_and_proof` on the network.
 (We define the following for convenience -- `aggregate_and_proof = signed_aggregate_and_proof.message` and `aggregate = aggregate_and_proof.aggregate`)
+- _[REJECT]_ The committee index is within the expected range -- i.e. `aggregate.data.index < get_committee_count_per_slot(state, aggregate.data.target.epoch)`.
 - _[IGNORE]_ `aggregate.data.slot` is within the last `ATTESTATION_PROPAGATION_SLOT_RANGE` slots (with a `MAXIMUM_GOSSIP_CLOCK_DISPARITY` allowance) --
   i.e. `aggregate.data.slot + ATTESTATION_PROPAGATION_SLOT_RANGE >= current_slot >= aggregate.data.slot`
   (a client MAY queue future aggregates for processing at the appropriate slot).
 - _[REJECT]_ The aggregate attestation's epoch matches its target -- i.e. `aggregate.data.target.epoch ==
   compute_epoch_at_slot(aggregate.data.slot)`
+- _[REJECT]_ The number of aggregation bits matches the committee size -- i.e.
+  `len(aggregate.aggregation_bits) == len(get_beacon_committee(state, aggregate.data.slot, aggregate.data.index))`.
+- _[REJECT]_ The aggregate attestation has participants --
+  that is, `len(get_attesting_indices(state, aggregate.data, aggregate.aggregation_bits)) >= 1`.
```

Comment:
this rule is a less strict version of the rule on 444 (exactly 1 participant), i wonder if it makes sense to collapse those into a single rule where we reject an aggregation that has less than 2 participants...

For reference
```
_[REJECT]_ The attestation is unaggregated --
  that is, it has exactly one participating validator (`len([bit for bit in attestation.aggregation_bits if bit]) == 1`, i.e. exactly 1 bit is set).
```
is the other rule I'm referring to.

Reply:
The block proposer is not necessarily subscribed to all individual attestation subnets. In case that there is a subnet with only a single attestation, there still needs to be a way for aggregators to collect that and re-broadcast it on the global aggregates topic, so that the block proposer can see it and include it. It is technically possible for such an aggregate to only include a single attestation.